### PR TITLE
Add logic question to contact us form to avoid spamming

### PR DESCRIPTION
**Pull request by jyobhai**
https://www.ebi.ac.uk/panda/jira/browse/ENSWEB-6954

Add a simple logic to stop spamming contact us page

http://wp-np2-1e.ebi.ac.uk:9000/

### CODE REVIEW

File: modules/EnsEMBL/Web/Command/Help/SendEmail.pm [hunk 1]
```diff
@@ -34,16 +34,17 @@ sub process {
...
   my $url;
 
-  if ($hub->param('logic') != '9') {
-    throw exception('Form validation error', 'Answer to the logical question is wrong');
-  }
+  my $logic_value = $hub->param('logic');
+  $logic_value =~ s/^\s+|\s+$//g;
 
-  if ($hub->param('submit') eq 'Back') {
+  my $error_msg = $logic_value != 9 ? "Are you a human verification failed! Please try again." : "";
```
Comment (azangru) [inline modules/EnsEMBL/Web/Command/Help/SendEmail.pm:40]: "Are you a human verification failed" reads awkward. Perhaps, "You did not provide the correct response to the control question. Please try again."?

Comment (jyobhai) [reply modules/EnsEMBL/Web/Command/Help/SendEmail.pm:40]: response to the logical question? instead of control?

Comment (azangru) [reply modules/EnsEMBL/Web/Command/Help/SendEmail.pm:40]: "You did not provide the correct response to the logical question" doesn't sound right either 😞

File: modules/EnsEMBL/Web/Component/Help/Contact.pm [hunk 2]
```diff
@@ -90,7 +90,7 @@ sub content {
     'label'     => 'What is 5 + 4? ',
     'value'     => uri_unescape($hub->param('logic')) || '',
     'required'  => 1,
-    'helptip'     => 'Are you a human verification'
+    'helptip'     => 'Question to verify you are a human or bot'
```
Comment (azangru) [inline modules/EnsEMBL/Web/Component/Help/Contact.pm:93]: `Question to verify you are a human or bot` -> `Question to verify that you are a human` (user probably shouldn't read that we may consider them a bot)

File: modules/EnsEMBL/Web/Command/Help/SendEmail.pm [hunk 3]
```diff
@@ -25,6 +25,7 @@ use strict;
 use warnings;
 
 use EnsEMBL::Web::Mailer::Help;
+use EnsEMBL::Web::Exceptions;
```
Comment (azangru) [inline modules/EnsEMBL/Web/Command/Help/SendEmail.pm:28]: You aren't using this anywhere anymore, are you?

**Review by azangru — APPROVED** [review]
Response with the filled in form on incorrect captcha input works fantastically well!

**Pull request merged**